pyvista/core/pointset.py

```diff
@@ -394,6 +394,12 @@ class PolyData(_vtk.vtkPolyData, PointSet, PolyDataFilters):
         file as if it had this extension as opposed to the one in the
         file.
 
+    force_float : bool, optional
+        Casts the datatype to float32 if points datatype are
+        non-float.  Default ``True``. Set this to ``False`` to allow
+        non-float types, though this may lead to errors when rotating
+        datasets.
```

Comment:
See comment on the equivalent block for `vtk_points`. Whatever wording ends up there should work here.

Reply:
```suggestion
        Casts the datatype to float32 if points datatype are
        non-float.  Default ``True``. Set this to ``False`` to allow
        non-float types, though this may lead to intermediate float
        truncation when rotating datasets.
```